Demo/Sources/Model/MediaDescription.swift

```diff
@@ -13,21 +13,28 @@ enum MediaDescription {
     }
 
     private static var dateFormatter: DateFormatter = {
-        let dateFormatter = DateFormatter()
-        dateFormatter.timeZone = TimeZone(identifier: "Europe/Zurich")
-        dateFormatter.dateStyle = .long
-        dateFormatter.timeStyle = .none
-        dateFormatter.doesRelativeDateFormatting = true
-        return dateFormatter
+        let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(identifier: "Europe/Zurich")
+        formatter.dateStyle = .long
+        formatter.timeStyle = .none
+        formatter.doesRelativeDateFormatting = true
+        return formatter
+    }()
+
+    private static var minuteFormatter: DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
+        formatter.allowedUnits = .minute
+        formatter.unitsStyle = .short
+        formatter.zeroFormattingBehavior = .pad
```

Comment:
If you go in "Lists" > "TV Topics" > "RTS" > "Séries", you can have durations bigger than 60 min. For example, "L'Adieu" is displayed as "94 min" long.
Wouldn't it be more user friendly to display it as "1 hr, 34 min"?
```suggestion
        formatter.allowedUnits = [.minute, .hour]
        formatter.unitsStyle = .short
        formatter.zeroFormattingBehavior = [.pad, .dropAll]
```

Reply:
Same comment as above. We had the behavior you mention in Play a few years ago but we moved to minute display only since we don't have streams lasting more than a couple of hours. This rule was borrowed as is here.